specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-12-01-preview/examples/ApiManagementPortalSettingsCorsGet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
```

Comment:
2021-12-01-preview, here and  in all the examples

Reply:
Good catch!